src/ui/components/Queue.tsx

```diff
@@ -25,23 +28,39 @@ const formatDate = (ts: any) => {
     : format(ts, 'MM/dd/yyyy HH:mm:ss')
 }
 
-const Timestamp = ({ ts, prev }: { ts: any; prev?: any }) => {
+const Timestamp = ({
+  ts,
+  prev,
+}: {
+  ts: TimeStamp | null
+  prev?: TimeStamp | null
+}) => {
+  if (ts === null) {
+    // REVIEW: what should be the timestamp for null?
```

Comment:
Unsure. Open for suggestions 🙂

Reply:
I think it makes sense to understand when it comes as null and provide a matching message, or skip rendering altogether.

Reply:
Ok!